resteasy-core-spi/src/main/java/org/jboss/resteasy/spi/ResteasyProviderFactory.java

```diff
@@ -133,7 +133,7 @@ public static ResteasyProviderFactory newInstance()
       try
       {
          return (ResteasyProviderFactory) Thread.currentThread().getContextClassLoader()
-               .loadClass("org.jboss.resteasy.core.ResteasyProviderFactoryImpl").newInstance();
+               .loadClass("org.jboss.resteasy.core.providerfactory.ResteasyProviderFactoryImpl").newInstance();

```

Comment:
can we change the deprecated (since java 9)  call `.newInstance()` to `.getDeclaredConstructor().newInstance()` (and extend catch clause)?

Reply:
done